app/views/articles/search.html.erb

```diff
@@ -11,7 +11,7 @@
   <div class="articles-list" id="articles-list">
     <div id="banner-section"></div>
     <div class="flex-1 flex items-center justify-between">
-      <h2 class="fs-l fw-heavy">Results</h2>
+      <h2 class="fs-l fw-heavy pl-2">Results</h2>
```

Comment:
This will also affect the desktop version of the site, so I'd suggest slightly different approach here:

```suggestion
      <h2 class="fs-l fw-heavy p-2 m:p-0">Results</h2>
```

This would apply `.p-2` (`padding: var(--su-2)`) class by default, and we reach medium breakpoint (this is what `m:` prefix stands for), it would apply `.p-0` (`padding: 0;`)

Also, as you can see my suggestion is to use `p-` instead of `pl-` so we can apply padding on all sides instead only left - this particular area would definitely appreciate more room on all sides :)